config.toml

```diff
@@ -0,0 +1,11 @@
+[catalog]
```

Comment:
This type of config is only supported by Python 3.11

Reply:
Fixed, I re-added python 3.8 compatibility and tested it